packaging/distros/ubuntu/Dockerfile

```diff
@@ -34,17 +33,18 @@ ENV DEBIAN_FRONTEND noninteractive
 
 # hadolint ignore=DL3008,DL3015
 RUN apt-get -qq update && \
-    apt-get install -y -qq curl ca-certificates build-essential libsystemd-dev \
+    apt-get install -y -qq curl ca-certificates gcc-8 g++-8 libsystemd-dev \
```

Comment:
Is GCC 8 a non-default compiler then for this platform? 
Presumably the binaries built still run on a vanilla install? We cannot require a standard library update or similar so just want to check.

Reply:
The default compiler on this OS is GCC 7. The binaries should be compatible, because the version requirements for libc6 didn't change between GCC 7 and GCC 8. 
The dependencies are listed here:
GCC 7: https://packages.ubuntu.com/en/bionic/gcc-7
GCC 8: https://packages.ubuntu.com/en/bionic/gcc-8

Reply:
I'll do a quick sanity check that the packages can be installed and run on a vanilla container instance just to check.

Reply:
Yeah all looks good for me here @niedbalski if you want to review.

```shell
$ ./packaging/build.sh -d ubuntu/18.04
...
CPack: - package: /tmp/fluent-bit/build/td-agent-bit_2.0.0_amd64.deb generated.
CPack: - package: /tmp/fluent-bit/build/td-agent-bit_2.0.0_amd64-headers.deb generated.
CPack: - package: /tmp/fluent-bit/build/td-agent-bit-2.0.0-1.x86_64-headers-extra.deb generated.
+ echo

+ echo 'Package(s) generated at: /home/pat/github/fluent/fluent-bit/packaging/packages/ubuntu/18.04/2022-10-03-14_18_29/'
Package(s) generated at: /home/pat/github/fluent/fluent-bit/packaging/packages/ubuntu/18.04/2022-10-03-14_18_29/
+ echo
$ docker run --rm -it -v /home/pat/github/fluent/fluent-bit/packaging/packages/ubuntu/18.04/2022-10-03-14_18_29/td-agent-bit_2.0.0_amd64.deb:/tmp/td-agent-bit.deb ubuntu:18.04 /bin/sh -c 'apt-get update && apt-get install -y /tmp/td-agent-bit.deb && /opt/td-agent-bit/bin/td-agent-bit'
...
The following additional packages will be installed:
  krb5-locales libasn1-8-heimdal libgssapi-krb5-2 libgssapi3-heimdal libhcrypto4-heimdal libheimbase1-heimdal libheimntlm0-heimdal libhx509-5-heimdal libk5crypto3 libkeyutils1 libkrb5-26-heimdal
  libkrb5-3 libkrb5support0 libldap-2.4-2 libldap-common libpq5 libroken18-heimdal libsasl2-2 libsasl2-modules libsasl2-modules-db libsqlite3-0 libssl1.1 libwind0-heimdal libyaml-0-2
Suggested packages:
  krb5-doc krb5-user libsasl2-modules-gssapi-mit | libsasl2-modules-gssapi-heimdal libsasl2-modules-ldap libsasl2-modules-otp libsasl2-modules-sql
The following NEW packages will be installed:
  krb5-locales libasn1-8-heimdal libgssapi-krb5-2 libgssapi3-heimdal libhcrypto4-heimdal libheimbase1-heimdal libheimntlm0-heimdal libhx509-5-heimdal libk5crypto3 libkeyutils1 libkrb5-26-heimdal
  libkrb5-3 libkrb5support0 libldap-2.4-2 libldap-common libpq5 libroken18-heimdal libsasl2-2 libsasl2-modules libsasl2-modules-db libsqlite3-0 libssl1.1 libwind0-heimdal libyaml-0-2 td-agent-bit
0 upgraded, 25 newly installed, 0 to remove and 2 not upgraded.
...
Setting up libpq5:amd64 (10.22-0ubuntu0.18.04.1) ...
Setting up td-agent-bit (2.0.0) ...
Processing triggers for libc-bin (2.27-3ubuntu1.6) ...
Fluent Bit v2.0.0
* Copyright (C) 2015-2022 The Fluent Bit Authors
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2022/10/03 13:29:53] [ info] [fluent bit] version=2.0.0, commit=, pid=495
[2022/10/03 13:29:53] [ info] [storage] version=1.3.0, type=memory-only, sync=normal, checksum=disabled, max_chunks_up=128
[2022/10/03 13:29:53] [ info] [cmetrics] version=0.4.0
[2022/10/03 13:29:53] [ info] [sp] stream processor started
...
```